src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const rightRobot = {};
```

Comment:
```suggestion
  const inversedRobot = {};
```

Reply:
yes, inversedRobot sound better as a name for variable